src/ypd/servlets/project_servlet.py

```diff
@@ -77,7 +88,7 @@ def submit(self):
                 else:
                     project = Solicited()
                 project.post(title, description, current_user)
-                return redirect(url_for('IndexView:get',
+                return redirect(url_for('IndexView:view',
```

Comment:
This should be 'IndexView:get' as per my bugfix the other day

Reply:
Fixed it, not sure why it changed it as i never played with your code other than to copy paste.